portfolio/src/main/webapp/style.css

```diff
@@ -11,31 +11,33 @@ body {
   padding: 1%;
 }
 
-#intro-content {
-  margin-left: auto;
-  margin-right: auto;
-  text-align:center;
-  width: 650px;
-}
-
 #basic-profile {
    border: 1px solid black;
+   display: inline-block;
    padding: 10px;
 }
 
-#inner-content {
-   border: 1px solid black;
-   text-align: center;
+form {
+  display: inline-block;
```

Comment:
Is there a `form` introduced in this PR? If not, let's isolate this change to its related PR.

Reply:
ah ok-noted! updated!